packages/typescript/src/TypescriptMutator.ts

```diff
@@ -41,9 +45,21 @@ export class TypescriptMutator {
   }
 }
 
-const shouldNodeBeSkipped = (node: ts.Node): boolean => {
+const shouldNodeBeSkipped = (node: ts.Node, sourceFile: ts.SourceFile, excludedExpressions: string[] = []): boolean => {
+  if (excludedExpressions.length > 0) {
+    const nodeFirstLine = getNodeFirstLine(node, sourceFile);
```

Comment:
I don't think this setup will work.

Take the following Angular component for example:
```ts
@Component({
  selector: 'app-bank-account',
  inputs: ['bankName', 'id: account-id'],
  template: `
    Bank Name: {{ bankName }}
    Account Id: {{ id }}
  `
})
export class BankAccountComponent {
  bankName: string|null = null;
  id: string|null = null;
  normalizedBankName: string|null = null;
}
```

If I don't want to mutate any `@Component({})` decorators, I could use the following config:
```js
mutator: {
  name: 'typescript',
  excludedExpressions: [
    '@Component({'
  ]
}
```

However, I think this will still mutate every property inside of my `@Component` because they are on a different linenumber.